docs/cri/config.md

```diff
@@ -16,11 +16,18 @@ it is recommended to use the `systemd` driver on systemd-based hosts for complia
 [the "single-writer" rule](https://systemd.io/CGROUP_DELEGATION/) of cgroups.
 
 To configure containerd to use the `systemd` driver, set the following option in `/etc/containerd/config.toml`:
++ Before containerd 2.0
 ```toml
 version = 2
 [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
   SystemdCgroup = true
 ```
++ In containerd 2.0
+```toml
+version = 3
+[plugins.'io.containerd.cri.v1.runtime'.containerd.runtimes.runc.options]
```

Comment:
Is the difference between `"` and `'` intentional?

Reply:
Sort of yes. This quoting style corresponds to the output of the `/usr/local/bin/containerd config default` command in v2.0